src/navigation/TabNavigator.tsx

```diff
@@ -27,8 +28,11 @@ export default function TabNavigator() {
           tabBarIcon: ({ color }) => <Icon name="home" color={color} />,
           headerRight: () => (
             <Icon
-              name="info-circle"
-              onPress={() => navigation.navigate('Modal')}
+              name="sign-out"
+              onPress={async () => {
+                await AsyncStorage.removeItem('userToken');
```

Comment:
Honestly, I'm kind of questioning whether or not AsyncStorage is the best thing to use for this. Open to suggestions!

Reply:
Low priority, if AsyncStorage works I think we should stick with it (although I do get lots of warnings when I start up the app.